packages/datetime/src/datePicker.tsx

```diff
@@ -279,6 +323,10 @@ export class DatePicker extends AbstractComponent<IDatePickerProps, IDatePickerS
             Utils.safeInvoke(this.props.onChange, value, false);
         }
 
+        if (typeof this.props.dayPickerProps.onMonthChange === "function") {
```

Comment:
```tsx
CoreUtils.safeInvoke(this.props.dayPickerProps.onMonthChange, value);
```

Seems like we should also invoke this function from `handleYearSelectChange`, no?

Reply:
indeed